web3/pm.py

```diff
@@ -151,7 +154,7 @@ def _get_release_id(self, package_name: str, version: str) -> bytes:
         pass
 
     @abstractmethod
-    def _get_all_release_ids(self, package_name: str) -> Tuple[bytes]:
+    def _get_all_release_ids(self, package_name: str) -> Iterable[Tuple[bytes]]:
```

Comment:
This type  hint means `((b'arst',), (b'tsra'))`, i.e. an iterable of length-1 tuples with a bytes value.  Is that what you intended?